src/utils/store.js

```diff
@@ -4,8 +4,15 @@ const store = new Store()
 
 if (store.get('version') !== 4) {
   store.clear()
-  store.set('seenWelcome', false)
-  store.set('config', null)
+
+  // default config
+  store.set('config', {
+    type: 'go',
+    path: '',
+    flags: ['--migrate=true', '--routing=dhtclient'],
+    keysize: 4096
```

Comment:
```suggestion
    keysize: 2048
```

https://github.com/ipfs/go-ipfs/blob/481136b10585e0d98bc9a5ec76953f644b43232b/cmd/ipfs/init.go#L25

https://github.com/ipfs/go-ipfs/issues/1107

out of curiosity where did the 4096 value come from?

Reply:
@olizilla honestly no idea 😆